adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsClientTest.java

```diff
@@ -306,15 +326,32 @@ private void verifyFileExists(String filename) throws URISyntaxException {
     }
 
     assertThat(hdfsLocalCluster.getHdfsFormat(), is(true));
-    assertThat(
-        uri.toString(),
-        is(
-            String.format(
-                "http://localhost:%s/webhdfs/v1/%s?op=GETFILESTATUS", HDFS_PORT, filename)));
+    assertThat(uri.toString(), is(String.format("%s%s?op=GETFILESTATUS", BASE_URL, filename)));
```

Comment:
Can you use the `url` variable here?

Reply:
url doesn't include the `op` parameter like uri does. Uri is built off of url though

Reply:
Oh I meant in the `is` part of that line. Like `is(String.format("%s?op=GETFILESTATUS", url)`

Reply:
It depends what you're passing in for `filename` though. It won't work if it has a leading slash.

Reply:
Ah, got you